test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] As stated by the text you're written, this file should be ignored by git. Currently it's not being ignored by git. - Add `test.md` to `.gitignore` file, 
- then try removing the `test.md` file from your new GitHub repo branch.
- then push your local git repo again.